File: app_main/src/main/java/app/jw/mapable/gm/Info/ViewPagerAdapter.java
package app.jw.mapable.gm.Info;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentStatePagerAdapter;

import org.jetbrains.annotations.NotNull;

import java.util.ArrayList;

public class ViewPagerAdapter extends FragmentStatePagerAdapter {
    final ArrayList<Fragment> items = new ArrayList<>();

    public ViewPagerAdapter(FragmentManager fragmentManager, int behavior)
    {
        super(fragmentManager, behavior);
    }

    public void addItem(Fragment item)
    {
        items.add(item);
    }


    @NotNull
    public Fragment getItem(int position)
    {
        return items.get(position);
    }

    @Override
    public int getCount() {
        return items.size();
    }
}

File: app_main/src/main/java/app/jw/mapable/gm/Info/ViewPagerAdapter.kt
package app.jw.mapable.gm.Info

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentStatePagerAdapter
import java.util.*

class ViewPagerAdapter(fragmentManager: FragmentManager?, behavior: Int) :
    FragmentStatePagerAdapter(fragmentManager!!, behavior) {
    val items = ArrayList<Fragment>()
    fun addItem(item: Fragment) {
        items.add(item)
    }

    override fun getItem(position: Int): Fragment {
        return items[position]
    }

    override fun getCount(): Int {
        return items.size
    }
}